### ISSUE TITLE
Support for OVH API v2

### ISSUE THREAD

**Issue by flo-sch** [ovh/php-ovh-sms#13]
Hey,

I am already using Guzzle >= 6.2 for a project ;
And this SMS client locks the OVH wrapper to v1.1.1, which locks Guzzle < 6.

Would it please be possible to release even a dev version supporting OVH >= v2?

Thanks in advance :)

**Comment by dsferruzza** [comment]
Hello @flo-sch 

I made a PR to fix this; see #20.
I have no idea if this repo is still maintained though; it might not be merged soon...
In the mid-time, if you still have the problem, you can use my fork:

```json
{
    "name": "Example Application",
    "description": "This is an example of OVH SMS APIs SDK usage",
    "repositories": [
        {
            "type": "vcs",
            "url": "https://github.com/startup-palace/php-ovh-sms"
        }
    ],
    "require": {
        "ovh/php-ovh-sms": "dev-update-ovh-api"
    }
}
```

*This might interest @VincentCasse @Metrakit @marcw and @SakeADroid as well(?)*

Have a nice day!

**Comment by flo-sch** [comment]
Hey @dsferruzza,

Many thanks for this, I actually forgot this thread,
I switched to the more generic `ovh/ovh` API since.

I have no idea neither about the plans of OVH for this project in the future,
But I guess your fix will be able to help people anyway :)

Have a nice day too!
